test/black-box/address/AddressAddCommand.test.ts

```diff
@@ -123,10 +123,27 @@ describe('AddressAddCommand', () => {
             testData.zipCode
         ]);
         res.expectJson();
-        res.expectStatusCode(400);
+        res.expectStatusCode(404);
         expect(res.error.error.success).toBe(false);
-        // TODO: better errors
-        expect(res.error.error.message).toBe(`Request body is not valid`);
+        expect(res.error.error.message).toBe(`Invalid state.`);
+    });
+
+    test('Should fail to create Address because state is undefined', async () => {
+        const res = await testUtil.rpc(addressCommand, [addressAddCommand,
+            defaultProfile.id,
+            testData.title,
+            testData.firstName,
+            testData.lastName,
+            testData.addressLine1,
+            testData.addressLine2,
+            testData.city,
+            testData.state,
+            testData.country,
+        ]);
+        res.expectJson();
+        res.expectStatusCode(404);
+        expect(res.error.error.success).toBe(false);
+        expect(res.error.error.message).toBe(`Missing zipCode.`);
```

Comment:
expect(res.error.error.message).toBe(new MissingParamException(...

Reply:
It doesn't consist with tests before - we were copying the text. Also it will need to take message from new MissingParamException nevertheless as it's Exception type.